internal/environment/lab/lab.go

```diff
@@ -17,9 +17,17 @@ import (
 
 const defaultImageMEMMB = 4096
 
+var (
+	BeginnerType = 0
+	AdvancedType = 1
```

Comment:
Is it possible to use the already defined values of this, to prevent future mistakes where it is changed in one place :)

Reply:
Hmm Yes, but i think initially i created this because environments can have a type, and labs can have a type. But since these are directly correlated you are probably right!